collective/jsonmigrator/blueprints/properties.py

```diff
@@ -66,6 +66,8 @@ def __iter__(self):
                     # if object have a attribute equal to property, do nothing
                     continue
 
+                if ptype == 'string':
+                    pvalue = safe_unicode(pvalue).encode('utf-8')
```

Comment:
I would say it would be better to check `if pvalue` is a unicode before and then just call `.encode`, something like:
```
if isinstance(pvalue, unicode):
    pvalue = pvalue.encode('utf8')
```
￼Also if you are able to add a test it would be great.

Anyway I do not consider these changes to be required for a merge.
If you have time please update your PR, otherwise just merge it :)

Reply:
tests would be nice, but since there are no test in the module it would be a bigger task to begin creating the infrastructure.